packages/experiments/src/components/Card/Card.styles.ts

```diff
@@ -0,0 +1,51 @@
+import { ICardComponent, ICardStylesReturnType, ICardTokenReturnType } from './Card.types';
+import { getGlobalClassNames } from '../../Styling';
+
+const GlobalClassNames = {
+  root: 'ms-Card',
+  stack: 'ms-Card-stack'
+};
+
+export const CardTokens: ICardComponent['tokens'] = (props, theme): ICardTokenReturnType => [];
+
+export const CardStyles: ICardComponent['styles'] = (props, theme, tokens): ICardStylesReturnType => {
```

Comment:
How does this feel? Better? Natural? Intuitive?

Reply:
Maybe not the best person to answer since I do have a little more background on this but I did think that it was easier to ramp up on this component, even though it was completely new, than when I tried to ramp up on converting the `Toogle` component to experiments.